### ISSUE TITLE
It this list correct?

### ISSUE THREAD

**Issue by stevenevans-j2** [dotnet/docs#5829]
I am using the package Microsoft.Build.Utilities.Core (the latest version 15.7.179) it has version past 4.5., But not 4.7.2 which i was looking for. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 681c3fa1-04b3-5c99-6122-ac94607cacc4
* Version Independent ID: 32af7ce5-ab2f-10bf-b7cf-87a2344c4d77
* Content: [TargetDotNetFrameworkVersion Enum (Microsoft.Build.Utilities)](https://docs.microsoft.com/en-us/dotnet/api/microsoft.build.utilities.targetdotnetframeworkversion?view=netframework-4.7.2)
* Content Source: [xml/Microsoft.Build.Utilities/TargetDotNetFrameworkVersion.xml](https://github.com/dotnet/dotnet-api-docs/blob/master/xml/Microsoft.Build.Utilities/TargetDotNetFrameworkVersion.xml)
* Product: **.net**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **susanno**

**Comment by mairaw** [comment]
Yes, the list on the docs is correct @stevenevans-j2.

![image](https://user-images.githubusercontent.com/12971179/41050771-358c2f98-6969-11e8-9ec5-2544d3f20ebb.png)

@karelz can you help route this question (Microsoft.Build.Utilities namespace)? Thx!

**Comment by karelz** [comment]
Looks like part of msbuild: https://github.com/Microsoft/msbuild
@nguerrera @dsplaisted can you please help route it?

**Comment by dsplaisted** [comment]
It is not up-to-date.  Here is the latest source code: https://github.com/Microsoft/msbuild/blob/master/ref/Microsoft.Build.Utilities.Core/netstandard/Microsoft.Build.Utilities.Core.cs#L152

I think the issue is that the API docs are showing the APIs from Microsoft.Build.Utilities.v4.0.dll, which is not the current version of the MSBuild APIs.

**Comment by Mikejo5000** [comment]
It might be good to get some input from @mairaw for how to manage the version issues... Maybe we could (1) provide notes in the .NET APIs with links to the GitHub source code (current version) and/or (2) Reflect over the current APIs, make them available in the .NET API Browser like some of the Visual Studio extensibility APIs. For the latter solution, however, it's possible we would have to set up a new repo, which isn't really ideal.

**Comment by dsplaisted** [comment]
As a comparison, it looks like there aren't any Roslyn APIs on docs.microsoft.com.  Instead, there is http://source.roslyn.io/ for browsing its source.  So maybe we could just update the docs with a message that these APIs are outdated, with a link to where to browse the current ones.  We could set up a source browser site for MSBuild, or maybe we could just use fuget.org: https://www.fuget.org/packages/Microsoft.Build.Utilities.Core/15.7.179/netstandard2.0/lib/Microsoft.Build.Utilities.Core.dll/Microsoft.Build.Utilities/TargetDotNetFrameworkVersion#TargetDotNetFrameworkVersion

**Comment by mairaw** [comment]
> As a comparison, it looks like there aren't any Roslyn APIs on docs.microsoft.com.

That is not true. See https://docs.microsoft.com/en-us/dotnet/api/?view=roslyn-dotnet. It's configured in a separate repo.

Yes, I think you would need a new repo for that. However, I don't know if docs has the ability to show two different pages for the same API depending on which product you selected. @dend can you comment on that?

**Comment by dsplaisted** [comment]
I see.  I searched for an API from Roslyn and it didn't show up.  Now I see that was because the context I was searching for was probably in .NET 4.7.2.
